mm2src/kdf_walletconnect/src/lib.rs

```diff
@@ -304,7 +304,7 @@ impl WalletConnectCtxImpl {
 
         send_proposal_request(self, &topic, required_namespaces, optional_namespaces).await?;
 
-        Ok(url)
+        Ok((url, topic))
```

Comment:
Returning a hard-typed value like `CreateConnectionResponse` would be a bit nicer.

Reply:
tbh, i was reluctant creating a new type to encapsulate the outputs since it's used only in this very function and this function is only called in one place.
i can create a type (or alias) for it if u think it's important tho.

Reply:
We already have a type (`CreateConnectionResponse`), we just need to find a better name and location to use it on both sides.